Cargo.lock

```diff
@@ -1257,12 +1481,35 @@ dependencies = [
  "tracing-subscriber",
 ]
 
+[[package]]
+name = "rtnetlink"
+version = "0.16.0"
+source = "git+https://github.com/rust-netlink/rtnetlink.git#2423645faed3fdad55eb8ea9fe7ea8b0649e6b26"
```

Comment:
`cargo deny` also complains about this. I'm not sure how this ended up here, did you set the source manually?